### PR TITLE
Swift Docs

### PR DESCRIPTION
**Pull request by DavidFair**

### CODE REVIEW

File: source/Swift/IntroducingSwift.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
+.. _introducing_swift:
+
+=====
+Swift
```
Comment (DavidFair) [inline source/Swift/IntroducingSwift.rst]: This title should be `Introducing Swift`, currently the search shows Swift twice as it has grabbed this title

File: source/Swift/IntroducingSwift.rst [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+=====
+
+
+.. _swift_introduction:
+
+Introduction
```
Comment (DavidFair) [inline source/Swift/IntroducingSwift.rst]: Then drop this title as it's redundant

File: source/Swift/IntroducingSwift.rst [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+------------
+
+Swift is a highly available, distributed, eventually consistent object/blob store. Swift allows lots of data to be stored efficiently, safely, and cheaply. It's built for scale and optimized for durability, availability, and concurrency across the entire data set. Swift is ideal for storing unstructured data that can grow without bound.
```
Comment (DavidFair) [inline source/Swift/IntroducingSwift.rst]: What does `eventually consistent` mean. You'll need to explain this as that phrase == data loss from a user POV

Comment (DavidFair) [reply source/Swift/IntroducingSwift.rst]: This needs some rephrasing IMO:

> It's built for scale and optimized for durability, availability, and concurrency across the entire data set.

Great for us as cloud operators, what benefit does a user get?

> Swift is ideal for storing unstructured data that can grow without bound.

What sort of data is unstructured data? Maybe some examples of the sort of data a user would and wouldn't put onto this

File: source/Swift/PythonSDK.rst [hunk 4]
```diff
@@ -0,0 +1,442 @@
...
+
+
+Downloading an object's contents using an ``Object`` instance (in the form of either ``obj_1`` or ``obj_2``):
+
+.. code-block:: python
+
+    file_1 = conn.object_store.download_object(obj)
+
+
+Alternatively, downloading contents using a ``Response`` object:
```
Comment (DavidFair) [inline source/Swift/PythonSDK.rst]: `using the Response object from above`

File: source/Swift/PythonSDK.rst [hunk 5]
```diff
@@ -0,0 +1,442 @@
...
+.. code-block:: python
+
+    from swiftclient.client import Connection
+    from keystoneauth1 import session
+    from keystoneauth1.identity import v3
+
+    # Create a password auth plugin
+    auth = v3.Password(auth_url='https://openstack.stfc.ac.uk:5000/v3',
+                        username='example_user',
+                        password='example_passwprd',
```
Comment (DavidFair) [inline source/Swift/PythonSDK.rst]: Nit: `example_password`

**Review by DavidFair — COMMENTED** [review]
Some changes but overall the docs are really good, nice work :+1:

**Comment by ElliottKasoar** [comment]
Thanks for the suggestions. The intro could probably use revisiting again to add a bit more detail, but otherwise I think I've addressed everything.

**Comment by DavidFair** [comment]
I'm happy to approve this, but as I opened the PR from the fork Github see's this as my own PR
@apdibbo Are you okay to approve and merge on my behalf?

**Review by apdibbo — APPROVED** [review]
Reviewed by David

**Pull request merged**